tests/installation/partitioning_raid.pm

```diff
@@ -163,10 +163,9 @@ sub run {
             assert_screen 'partitioning-type';
             send_key 'alt-n';
         }
-        assert_screen 'partitioning-size';
+        assert_screen 'partition-size';
```

Comment:
Our current SLE ppc64le tests have needles with the tag `partitioning-size` so I would prefer to keep this and the following code as is.